docs/layouts/shortcodes/build_table.html

```diff
@@ -0,0 +1,42 @@
+<style type="text/css">
+  .tg  {border-collapse:collapse;border-spacing:0;}
+  .tg td{border-color:black;border-style:solid;border-width:1px;font-family:Arial, sans-serif;font-size:14px;
+    overflow:hidden;padding:10px 5px;word-break:normal;}
+  .tg th{border-color:black;border-style:solid;border-width:1px;font-family:Arial, sans-serif;font-size:14px;
+    font-weight:normal;overflow:hidden;padding:10px 5px;word-break:normal;}
+  .tg .tg-c3ow{border-color:black;text-align:center;vertical-align:top}
+  .tg .tg-0pky{border-color:black;text-align:left;vertical-align:top}
+  .tg .tg-dgl5{border-color:black;background-color:#FFF;text-align:left;vertical-align:top}
+  </style>
+  <table class="tg">
+  <thead>
+    <tr>
+      <th class="tg-c3ow" colspan="2"><span style="font-weight:bold">magic-modules - build commands</span></th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td class="tg-0pky"><span style="font-weight:bold">make</span><br>runs make build</td>
+      <td class="tg-0pky"><span style="font-weight:bold">make build</span><br>  Builds the terraform provider with both mmv1 and tpgtools. Copies all generated files as a merge with the provider in OUTPUT_PATH.</td>
+    </tr>
+    <tr>
+      <td class="tg-0pky"><span style="font-weight:bold">make upgrade-dcl</span><br> A command to upgrade the version of the dcl. Used within the terraform provider and tpgtools.</td>
+      <td class="tg-0pky"><span style="font-weight:bold">make validate_environment</span><br> A prestep to make build to ensure that the build environment is suitable for a build.</td>
```

Comment:
Since validate_environment is already part of `make terraform` we probably don't need to document it separately on this page. What would you think about recommending this in the "set up your environment" page instead of the doctor script?

I remember talking previously about the output path / version validation being part of this make command vs not - I don't remember the conclusion though!

Separate but related topic: What is the use case for having validate_environment as a standalone command vs. doing the check directly in the generation command & having the doctor script? The main benefit I see immediately is that `make validate_environment` would let us change the implementation easily later. But it is a bit long - what about `make doctor`? (Analogous to [`brew doctor`](https://docs.brew.sh/Troubleshooting) for homebrew.)

Reply:
ack this makes sense.  I'll omit in favor of just running doctor (already supported). Sure updating validate_environment makes sense to me!